README.rst

```diff
@@ -19,10 +19,53 @@ on Python 2.6 or 3.3 will need to upgrade their version of Python or pin the
 version of Botocore in use prior to 01/10/2020. For more information, see
 this `blog post <https://aws.amazon.com/blogs/developer/deprecation-of-python-2-6-and-python-3-3-in-botocore-boto3-and-the-aws-cli/>`__.
 
+Getting Started
+---------------
+Assuming that you have Python and ``virtualenv`` installed, set up your environment and install the required dependencies like this or you can install the library using ``pip``:
 
-Documentation
--------------
-Documentation for ``botocore`` can be found  `here <https://botocore.amazonaws.com/v1/documentation/api/latest/index.html>`__.
+.. code-block:: sh
+
+    $ git clone https://github.com/boto/botocore.git
+    $ cd botocore
+    $ virtualenv venv
+    ...
+    $ . venv/bin/activate
+    $ pip install -r requirements.txt
+    $ pip install -e .
+
+.. code-block:: sh
+
+    $ pip install botocore
+    
+Using Botocore
+~~~~~~~~~~~~~~
+After installing botocore 
+
+Next, set up credentials (in e.g. ``~/.aws/credentials``):
+
+.. code-block:: ini
+
+    [default]
+    aws_access_key_id = YOUR_KEY
+    aws_secret_access_key = YOUR_SECRET
+
+Then, set up a default region (in e.g. ``~/.aws/config``):
+
+.. code-block:: ini
+
+   [default]
+   region=us-east-1
+    
+Other credentials configuration method can be found `here <https://boto3.amazonaws.com/v1/documentation/api/1.9.42/guide/configuration.html>`__
```

Comment:
Let's update this link to the `latest` instead of `1.9.42`.

Reply:
Updated the link to reflect latest version(1.14.20)